libs/notifications/package.json

```
@@ -0,0 +1,51 @@
+{
+  "name": "@novu/notifications",
+  "version": "1.0.0",
+  "description": "Novu notification templates and workflows",
+  "main": "build/main/index.js",
+  "typings": "build/main/index.d.ts",
+  "module": "build/module/index.js",
+  "private": true,
+  "license": "MIT",
+  "keywords": [],
+  "scripts": {
+    "build": "run-p build:*",
+    "build:main": "tsc -p tsconfig.json",
+    "build:module": "tsc -p tsconfig.module.json",
+    "fix": "run-s fix:*",
+    "fix:prettier": "prettier \"src/**/*.ts\" --write",
+    "fix:lint": "eslint src --ext .ts --fix",
+    "test": "jest --passWithNoTests",
+    "lint": "eslint src",
+    "test:watch": "jest --watch",
```

Preparing for better test tooling 😉

```suggestion
    "test": "vitest --passWithNoTests",
    "lint": "eslint src",
    "test:watch": "vitest --watch",
```

---

💪